src/orquestra/sdk/_ray/_dag.py

```diff
@@ -158,6 +158,7 @@ def _make_ray_dag_node(
     ray_kwargs: t.Mapping[str, t.Any],
     pos_unpack_specs: t.Sequence[PosArgUnpackSpec],
     kw_unpack_specs: t.Sequence[KwArgUnpackSpec],
+    project_dir,
```

Comment:
Our code in this file is already difficult to follow. Please add the type annotation and describe what it is for in the docstring
```suggestion
    project_dir: t.Optional[Path],
```